test/functional/tests/lazy_writes/cleaning_policy/acp_params.py

```diff
@@ -0,0 +1,266 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+import time
+import random
+from datetime import timedelta
+
+from test_utils.size import Size, Unit
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools import fs_utils
+from test_tools.fio.fio import Fio
+from test_utils.os_utils import kill_all_io
+from test_tools.fio.fio_param import ReadWrite, IoEngine
+from api.cas import casadm
+from api.cas.cache_config import (
+    CacheMode,
+    CacheModeTrait,
+    CleaningPolicy,
+    FlushParametersAcp,
+    CacheLineSize,
+    Time,
+)
+from test_tools.blktrace import BlkTrace, BlkTraceMask, ActionKind, RwbsKind
+
+
+@pytest.mark.parametrize(
+    "cache_line_size",
+    [CacheLineSize.LINE_4KiB, CacheLineSize.LINE_16KiB, CacheLineSize.LINE_64KiB],
+)
+@pytest.mark.parametrize(
+    "cache_mode",
+    [
+        mode
+        for mode in CacheMode
+        if CacheModeTrait.LazyFlush & CacheMode.get_traits(mode)
```

Comment:
Change LazyFlush to LazyWrites, as the name changes in framework a few PRs ago. And also you could change "&" to "in" as this is more elegant use of trait check.